src/Common/Version.props

```diff
@@ -1,7 +1,7 @@
 <Project>
   <PropertyGroup>
     <AssemblyVersion>1.55.0</AssemblyVersion>
-    <PackageVersion>$(AssemblyVersion)-beta-4</PackageVersion>
+    <PackageVersion>$(AssemblyVersion)</PackageVersion>
     <DriverVersion>1.55.0-beta-1756314050000</DriverVersion>
```

Comment:
@mxschmitt is it intentional that that the 1.55.0 release is using a beta DriverVersion? We try and align our playwright node and dotnet versions to the DriverVersion so we aren't mixing browsers used in different tests.

Reply:
Yes, this is intentional, the internal driver version can be a beta which means its still from the `release-*` branch.